src/common/runnable/run-many-sync-for.ts

```diff
@@ -19,7 +19,7 @@ type RunSync<Param> = (parameter: Param) => void;
 export type RunManySync = <Param>(injectionToken: InjectionToken<RunnableSync<Param>, void>) => RunSync<Param>;
 
 function runCompositeRunnableSyncs<Param>(param: Param, composite: Composite<RunnableSync<Param>>) {
-  assert(!((composite.value.run(param) as any) instanceof Promise), "Cannot be an async function for runnable sync");
+  assert(!((composite.value.run(param) as unknown) instanceof Promise), `Cannot be an async function for runnable sync`);
```

Comment:
Technical unit test for integral technical function is missing. As policy, our team could **at least** match the quality of code we encounter. Eg. this function is otherwise unit tested, so let's not damage it with addition of non-tested stuff.